src/common/rbac/project/util.go

```diff
@@ -21,12 +21,80 @@ import (
 var (
 	// subresource policies for public project
 	publicProjectPolicies = []*rbac.Policy{
-		{Resource: ResourceImage, Action: ActionPull},
+		{Resource: ResourceSelf, Action: ActionRead},
+
+		{Resource: ResourceRepository, Action: ActionList},
+		{Resource: ResourceRepository, Action: ActionPull},
+
+		{Resource: ResourceHelmChart, Action: ActionDownload},
+		{Resource: ResourceHelmChart, Action: ActionList},
+
+		{Resource: ResourceHelmChartVersion, Action: ActionDownload},
+		{Resource: ResourceHelmChartVersion, Action: ActionRead},
+		{Resource: ResourceHelmChartVersion, Action: ActionList},
 	}
 
-	// subresource policies for system admin visitor
-	systemAdminProjectPolicies = []*rbac.Policy{
-		{Resource: ResourceAll, Action: ActionAll},
+	// all policies for the projects
+	allPolicies = []*rbac.Policy{
+		{Resource: ResourceSelf, Action: ActionRead},
```

Comment:
Can we use the `Resource: *, Action *` to represent the all?

Reply:
all policies for project shoud be list here, so that in `/users/current/permissions` we can use `security.Context` Can method to check user's permissions